pkg/virt-handler/vm.go

```diff
@@ -376,10 +377,29 @@ func (d *VMHandlerDispatch) injectDiskAuth(vm *v1.VM) (*v1.VM, error) {
 	return vm, nil
 }
 
+func InsertBridgeVIF(vm *v1.VM, vif *network.VirtualInterface) (*v1.VM, error) {
+	vmCopy := &v1.VM{}
+	model.Copy(vmCopy, vm)
+
+	for idx, inter := range vmCopy.Spec.Domain.Devices.Interfaces {
+		fmt.Println("interface: ", inter)
+		if inter.Type == "network" {
```

Comment:
we'll have to think about how to represent Pod vs Host networking on the VM api side.

Reply:
Yep, thanks. I wanted to bring it up as well.